deploy/jbossas/kit/jboss-wf9/org/apache/lucene/5.3.1/module.xml

```diff
@@ -23,8 +23,16 @@
         <resource-root path="lucene-queries-${version.org.apache.lucene}.jar" />
         <resource-root path="lucene-queryparser-${version.org.apache.lucene}.jar" />
         <resource-root path="lucene-sandbox-${version.org.apache.lucene}.jar" />
+	<resource-root path="lucene-join-${version.org.apache.lucene}.jar" />
+	<resource-root path="lucene-highlighter-${version.org.apache.lucene}.jar" />
+	<resource-root path="lucene-memory-${version.org.apache.lucene}.jar" />
+	<resource-root path="lucene-suggest-${version.org.apache.lucene}.jar" />
+	<resource-root path="lucene-misc-${version.org.apache.lucene}.jar" />
+	<resource-root path="lucene-grouping-${version.org.apache.lucene}.jar" />
+	<resource-root path="lucene-spatial-${version.org.apache.lucene}.jar" />
```

Comment:
Ah, never mind. I guess the Elastic Search client library probably requires them all, even if we don't directly use those aspects of Lucene.